core/src/main/scala/org/genivi/sota/core/VehicleUpdatesResource.scala

```diff
@@ -183,7 +183,9 @@ class VehicleUpdatesResource(db : Database, resolverClient: ExternalResolverClie
       put {
         path("installed") { updateInstalledPackages(vin) } ~
         path("order") { setInstallOrder(vin) } ~
-        (extractUuid & path("cancelupdate") ) { uuid => cancelUpdate(vin, uuid) }
+        (extractUuid & path("cancelupdate") ) { uuid =>
+          namespaceExtractor { ns => cancelUpdate(ns, vin, uuid) }
```

Comment:
A workaround is feasible, to allow ota to continue using `cancelUpdate` without namespace, at the cost of accepting in that method (sota-side) an `Option[Namespace]`.

Or, would you say it's better to change ota to have a namespace ready on that spot?